onnxruntime/test/perftest/command_args_parser.cc

```diff
@@ -166,15 +161,19 @@ namespace perftest {
       "\t-X [Enable onnxruntime-extensions custom ops]: Registers custom ops from onnxruntime-extensions. "
       "onnxruntime-extensions must have been built in to onnxruntime. This can be done with the build.py "
       "'--use_extensions' option.\n"
+      "\t--plugin_ep_libs [registration names and libraries] Specifies a list of plugin execution provider(EP) registration names and their corresponding shared libraries to register.\n"
```

Comment:
>\t--plugin_ep_libs [ [](http://example.com/codeflow?start=7&length=20)

The -e option seems to imply that we can only test with one ep at a time. Here it is a collection. How does this match one with another?

Reply:
how are the EP options specified with `-i` interpreted if there is more than one EP?

Reply:
i added `--plugin_eps` and `--plugin_ep_options` options to support EPs and their provider options.
Please see the PR description for details and usage.
Let me know if you have other suggestions.